lib/model/rwfolder.go

```diff
@@ -690,20 +692,9 @@ func (f *rwFolder) deleteDir(file protocol.FileInfo, matcher *ignore.Matcher) {
 		f.newError(file.Name, err)
 		return
 	}
-	// Delete any temporary files lying around in the directory
-	dir, _ := os.Open(realName)
-	if dir != nil {
-		files, _ := dir.Readdirnames(-1)
-		for _, dirFile := range files {
-			fullDirFile := filepath.Join(file.Name, dirFile)
-			if defTempNamer.IsTemporary(dirFile) || (matcher != nil && matcher.Match(fullDirFile).IsDeletable()) {
-				os.RemoveAll(filepath.Join(f.dir, fullDirFile))
-			}
-		}
-		dir.Close()
-	}
 
-	err = osutil.InWritableDir(os.Remove, realName)
+	err = deletedir(f.dir, file, matcher)
```

Comment:
The folder type is correctly displayed by model.StartFolder():
`[YKC4E] 14:38:32 INFO: Restarted folder wvd4j-ksyym (receiveonly)`

I've tried making the change you suggested, but I can't get it to work. The embedding of the other type only partially works. Might simply be my limited understanding of golang...